packages/devextreme/js/__internal/scheduler/workspaces/m_work_space.ts

```diff
@@ -909,6 +909,38 @@ class SchedulerWorkSpace extends Widget<WorkspaceOptionsInternal> {
     }
   }
 
+  updateHeaderPanelScrollbarPadding() {
+    if (hasWindow() && this._$headerPanelContainer) {
+      const scrollbarWidth = this._getScrollbarWidth();
+      this._$headerPanelContainer.css('paddingRight', `${scrollbarWidth}px`);
+    }
+  }
+
+  _getScrollbarWidth() {
+    if (!this._dateTableScrollable) {
+      return 0;
+    }
+
+    const $container = this._dateTableScrollable.$element().find('.dx-scrollable-container');
+    if (!$container.length) {
+      return 0;
+    }
+
+    const containerElement = $container.get(0);
+    if (!containerElement) {
+      return 0;
+    }
+
+    const hasVerticalScrollbar = containerElement.scrollHeight > containerElement.clientHeight;
```

Comment:
Also Scrollable has `scrollHeight` and `clientHeight` methods, that we could possible use here.

https://github.com/DevExpress/DevExtreme/blob/45525faa3bd834615ab7406c114121374fce698e/packages/devextreme/js/__internal/ui/scroll_view/scrollable.ts#L493

Reply:
If it wouldn't be I hard, I would suggest to add Scrollable type to `_dateTableScrollable`